src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -59,9 +63,93 @@ constexpr int8_t kDefaultDeadBand                 = 25; // 2.5C is the default
 #define FEATURE_MAP_SCH 0x08
 #define FEATURE_MAP_SB 0x10
 #define FEATURE_MAP_AUTO 0x20
+#define FEATURE_MAP_LTNE 0x40
```

Comment:
This is unused...  And why are we bothering, when we already have a bitmap code-generated for us?